### PR TITLE
makes npm targets work on Windows

### PR DESCRIPTION
**Pull request by jbuffin**
- environment variables with `cross-env`
- `&` works differently on windows so use `concurrently` instead
- `rm -rf` is Unix only so use `rimraf` library

### CODE REVIEW

**Comment by jbuffin** [comment]
https://github.com/mz026/universal-redux-template/issues/14

**Comment by jbuffin** [comment]
I am having a problem running the test script as well on windows. If I remove the single quotes around the target regex, it works like this:

`cross-env NODE_ENV=test cross-env NODE_PATH=./app mocha --compilers js:babel-register -r app/spec/support/setup.mocha.js --recursive app/spec/**/*.test.js -w`

I will submit a separate PR for that.

File: package.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
   "version": "0.0.0",
   "description": "Universal Redux Template",
   "scripts": {
-    "start": "NODE_PATH=./app nodemon app/server & node webpack.server.js & gulp css:watch",
-    "test": "NODE_ENV=test NODE_PATH=./app mocha --compilers js:babel-register -r app/spec/support/setup.mocha.js --recursive 'app/spec/**/*.test.js' -w",
+    "start": "cross-env NODE_PATH=./app concurrent --kill-others \"nodemon app/server\" \"node webpack.server.js\" \"npm run css:watch\"",
```
Comment (mz026) [inline package.json]: The CSS job doesn't run since there's no npm script named `css:watch`. It should be `gulp css:watch`

File: package.json [hunk 2]
```diff
@@ -3,10 +3,10 @@
...
+    "test": "cross-env NODE_ENV=test cross-env NODE_PATH=./app mocha --compilers js:babel-register -r app/spec/support/setup.mocha.js --recursive 'app/spec/**/*.test.js' -w",
     "webpack": "webpack",
-    "build": "rm -rf dist/; npm run webpack; gulp css:build",
+    "build": "rimraf dist/ & npm run webpack & gulp css:build",
```
Comment (mz026) [inline package.json]: Using `&` to connect npm jobs here makes the `build` job hang there until receiving user input (like pressing `enter`). I think we can leave it running sequentially by concatenating the jobs with `;`

**Comment by mz026** [comment]
@jbuffin Thanks a lot! I left some comments on the commits. Also, could you please update `npm-shringwrap.json` accordingly?

File: package.json (hunk 1 above)
Comment (jbuffin) [inline package.json]: good catch, I totally missed that one

File: package.json (hunk 2 above)
Comment (jbuffin) [inline package.json]: I changed it to `&&` could you check to see if that works for you? `;` will not work in windows

Comment (mz026) [inline package.json]: yes! `&&` works fine on my linux machine. Let's go with `&&`

**Comment by jbuffin** [comment]
@mz026 is there anything else you need on this?

**Comment by mz026** [comment]
@jbuffin for the `npm-shrinkwrap.json`, could you please include `devDepencencies` in it? The `postinstall`, which builds code after deployment, needs the `devDependencies` to build stuff.

**Comment by mz026** [comment]
@jbuffin any update on this?

**Comment by jbuffin** [comment]
Yes sorry, I got busy yesterday and didn't get a chance to rerun shrinkwrap with `--dev`. I will get that sometime today and push it.

**Comment by mz026** [comment]
no problem! thanks a lot! :+1:

**Comment by jbuffin** [comment]
@mz026 I had to update one of the packages in order to get shrinkwrap to run correctly. All should be good now.

**Comment by mz026** [comment]
@jbuffin thanks!!

**Pull request merged**